rainier-core/src/main/scala/com/stripe/rainier/core/Sample.scala

```diff
@@ -0,0 +1,29 @@
+package com.stripe.rainier.core
+
+import com.stripe.rainier.compute._
+import com.stripe.rainier.sampler._
+
+case class Sample[+T](chains: List[List[Array[Double]]],
```

Comment:
Would it be better to favor `Seq` over `List` in these APIs since there's nothing particular about `List` that they need?

Reply:
I think at Stripe we tend to discourage `Seq` in favor of concrete types in most situations.